styles/variables-base.less

```diff
@@ -45,7 +45,7 @@
 
 // Common
 // ---------------------------------------
-@agate-base-font-family: "MuseoSans", sans-serif;
+@agate-base-font-family: "Agate", sans-serif;
```

Comment:
previously, in fonts.less, the "MuseoSans" font-family was defined. Since now Agate can use multiple fonts (Noto or Museo), I modified the name of the defined font-family to "Agate", to be neutral. The same situation is in the other UI libraries